reducer-todo-new/src/components/ToDoCard.js

```diff
@@ -0,0 +1,33 @@
+
+import React from 'react';
+import { Card } from 'semantic-ui-react'
+import TodoList from './TodoList'
+import ToDoForm from './TodoForm'
+
+function ToDoCard(props) {
+    
+    const { state } = props 
+    const { dispatch } = props
```

Comment:
you can actually destructure both items in 1 line.

`const { state, dispatch } = props`